code/modules/mob/living/carbon/taste.dm

```diff
@@ -77,15 +75,13 @@ calculate text size per text.
 			temp_text = "lukewarm"
 		if(T0C + 25 to T0C + 40)
 			temp_text = "warm"
-		if(T0C + 40 to T0C + 70)
+		if(T0C + 40 to T0C + 100)
 			temp_text = "hot"
-		if(T0C + 70 to T0C + 90)
+		if(T0C + 100 to T0C + 120)
```

Comment:
Above the boiling point of water is much more than "scolding hot", I would say, and regardless it's "scalding hot". Why were the numbers adjusted like this?

Reply:
because the perfect temperature of coffee is considered "scolding hot" so i adjusted it

this is a weird thing to complain about